.github/workflows/nightly.yml

```diff
@@ -0,0 +1,21 @@
+name: nightly
+on:
+  schedule:
+    - cron: '04 05 * * *'
+
+jobs:
+  deny:
+    name: cargo deny --all-features check
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
+      - uses: moonrepo/setup-rust@v1
```

Comment:
Internal policy says that for actions not in google orgs, you need to use a commit, not a tag.

Same in a few other places.

Reply:
fixed